bindings/python/tests/test_sqlalchemy.py

```diff
@@ -69,11 +69,11 @@ def test_create_index(session: Session, index_name: str, index_setting: str):
         index_name,
         Document.embedding,
         postgresql_using="vectors",
-        postgresql_with={"options": index_setting},
+        postgresql_with={"options": f"$${index_setting}$$"},
         postgresql_ops={"embedding": "l2_ops"},
     )
     index.create(session.bind)
-    session.rollback()
+    session.commit()
```

Comment:
Why does this action need to be committed while others can be rollback?

Reply:
I guess some problems emerge when the index exists. I'm still trying.